.github/workflows/black-fix.yml

```
@@ -39,7 +39,7 @@ jobs:
       run: |
         pwd
         ls
-        $HOME/.local/bin/black tests/ tools/ $(grep '^#.*python.*$' bin/ tests/ tools/ src/ -Ir|cut -d: -f1|grep -v _boutcore_build)
+        $HOME/.local/bin/black tests/ tools/ $(grep '^#.*python.*$' bin/ tests/ tools/ src/ -Ir|cut -d: -f1|grep -v _boutpp_build)
```

This picks up `# Install python packages` in `bin/bout-build-deps.sh`

I tried using `file`, which mostly gets it correct, but there are still a few false positives and negatives

Maybe:

```suggestion
        $HOME/.local/bin/black tests/ tools/ $(grep '^#.*python.*$' bin/ tests/ tools/ src/ -Ilr | grep -v _boutpp_build)
```